Model already exists: ../sfd-main/src/cifar10/edm-cifar10-32x32-uncond-vp.pkl
Loading the pre-trained diffusion model from "../sfd-main/src/cifar10/edm-cifar10-32x32-uncond-vp.pkl"...
cherish
4
torch.Size([12])
teacher now
torch.Size([3])
/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/distributed/distributed_c10d.py:4631: UserWarning: No device id is provided via `init_process_group` or `barrier `. Using the current device set by the user. 
  warnings.warn(  # warn only once
Total parameters in U-Net:      60417164
Setting up optimizer...
[rank0]: Traceback (most recent call last):
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/train.py", line 155, in <module>
[rank0]:     main()
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 1161, in __call__
[rank0]:     return self.main(*args, **kwargs)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 1082, in main
[rank0]:     rv = self.invoke(ctx)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 1443, in invoke
[rank0]:     return ctx.invoke(self.callback, **ctx.params)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/click/core.py", line 788, in invoke
[rank0]:     return __callback(*args, **kwargs)
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/train.py", line 150, in main
[rank0]:     training_loop.training_loop(**c)
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/training/training_loop.py", line 227, in training_loop
[rank0]:     optimizer = dnnlib.util.construct_class_by_name(params=net.parameters(), **optimizer_kwargs) # subclass of torch.optim.Optimizer
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/dnnlib/util.py", line 302, in construct_class_by_name
[rank0]:     return call_func_by_name(*args, func_name=class_name, **kwargs)
[rank0]:   File "/teamspace/studios/this_studio/sfd-main/dnnlib/util.py", line 297, in call_func_by_name
[rank0]:     return func_obj(*args, **kwargs)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/optim/adam.py", line 100, in __init__
[rank0]:     super().__init__(params, defaults)
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/optim/optimizer.py", line 369, in __init__
[rank0]:     self.add_param_group(cast(dict, param_group))
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_compile.py", line 46, in inner
[rank0]:     import torch._dynamo
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/__init__.py", line 13, in <module>
[rank0]:     from . import config, convert_frame, eval_frame, resume_execution
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/convert_frame.py", line 52, in <module>
[rank0]:     from torch._dynamo.symbolic_convert import TensorifyState
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/symbolic_convert.py", line 57, in <module>
[rank0]:     from . import (
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/trace_rules.py", line 32, in <module>
[rank0]:     from .variables import (
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/variables/__init__.py", line 19, in <module>
[rank0]:     from .base import VariableTracker
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/variables/base.py", line 581, in <module>
[rank0]:     from . import builder
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/variables/builder.py", line 241, in <module>
[rank0]:     from .torch import (
[rank0]:   File "/home/zeus/miniconda3/envs/cloudspace/lib/python3.9/site-packages/torch/_dynamo/variables/torch.py", line 176, in <module>
[rank0]:     torch._dynamo.external_utils.is_compiling: True,
[rank0]: AttributeError: partially initialized module 'torch._dynamo' has no attribute 'external_utils' (most likely due to a circular import)
